code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -5,8 +5,11 @@ What are the archived variables for?
 */
 #define MINIMUM_HEAT_CAPACITY	0.0003
 #define MINIMUM_MOLE_COUNT		0.01
-#define QUANTIZE(variable)		(round(variable,0.0000001))/*I feel the need to document what happens here. Basically this is used to catch most rounding errors, however it's previous value made it so that
+#define MINIMUM_MOLE_MANIPULATION_COUNT		0.001
+#define QUANTIZE(variable)		(round(variable, MINIMUM_MOLE_MANIPULATION_COUNT))/*I feel the need to document what happens here. Basically this is used to catch most rounding errors, however it's previous value made it so that

```

Comment:
Have you tested this? I am almost certain that this will break fusion and high temperature gas movement even more then it is now.

Reply:
i test it now. Has problems with minimum mole transfer. it create negative moles that bypas garbage somewhere,
try place logging.

Reply:
If you can figure that out please let me know, I've been experiencing a similar issue.

Reply:
as i see now
/obj/item/organ/lungs/proc/check_breath() 
add zero gases to mix to use it in formulas
but `breath.garbage_collect()` on end of proc dont clear it propertly.

Reply:
Do you have the info on the archived molar count when it's failing to gc?

Reply:
second number in check_breath() is ARCHIVE
```
		gasstring = ""
		for(var/gasid in breath.gases)
			gasstring += "[gasid] [breath.gases[gasid][MOLES]] [breath.gases[gasid][ARCHIVE]], "
		to_chat(world, "## TESTING: check_breath() b [gasstring]")
```
i test in https://github.com/Dennok/Dennok_tgstation/tree/gas-twixes build

Reply:
OMG, i screwed up
i remove `cached_gases -= id` from garbage_collect()
in the last check for sure

Reply:
i fix garbage collect and now has right breath test
```
## TESTING: check_breath() Starting gas /datum/gas/oxygen 0.0044 0, /datum/gas/nitrogen 0.0164 0, /datum/gas/carbon_dioxide 0.001 0, 
## TESTING: check_breath() b /datum/gas/oxygen 0 0, /datum/gas/nitrogen 0.0164 0, /datum/gas/carbon_dioxide 0.0054 0, /datum/gas/plasma 0 0, /datum/gas/nitrous_oxide 0 0, /datum/gas/bz 0 0, /datum/gas/tritium 0 0, /datum/gas/nitryl 0 0, /datum/gas/pluoxium 0 0, /datum/gas/stimulum 0 0, /datum/gas/freon 0 0, 
## TESTING: garbage_collect(breath) 0 /datum/gas/oxygen 0 0
## TESTING: garbage_collect(breath) removed /datum/gas/oxygen
## TESTING: garbage_collect(breath) 0.0164 /datum/gas/nitrogen 0.0164 0
## TESTING: garbage_collect(breath) 0.0054 /datum/gas/carbon_dioxide 0.0054 0
## TESTING: garbage_collect(breath) 0 /datum/gas/plasma 0 0
## TESTING: garbage_collect(breath) removed /datum/gas/plasma
## TESTING: garbage_collect(breath) 0 /datum/gas/nitrous_oxide 0 0
## TESTING: garbage_collect(breath) removed /datum/gas/nitrous_oxide
## TESTING: garbage_collect(breath) 0 /datum/gas/bz 0 0
## TESTING: garbage_collect(breath) removed /datum/gas/bz
## TESTING: garbage_collect(breath) 0 /datum/gas/tritium 0 0
## TESTING: garbage_collect(breath) removed /datum/gas/tritium
## TESTING: garbage_collect(breath) 0 /datum/gas/nitryl 0 0
## TESTING: garbage_collect(breath) removed /datum/gas/nitryl
## TESTING: garbage_collect(breath) 0 /datum/gas/pluoxium 0 0
## TESTING: garbage_collect(breath) removed /datum/gas/pluoxium
## TESTING: garbage_collect(breath) 0 /datum/gas/stimulum 0 0
## TESTING: garbage_collect(breath) removed /datum/gas/stimulum
## TESTING: garbage_collect(breath) 0 /datum/gas/freon 0 0
## TESTING: garbage_collect(breath) removed /datum/gas/freon
## TESTING: check_breath() b /datum/gas/nitrogen 0.0164 0, /datum/gas/carbon_dioxide 0.0054 0, 
```